app/src/main/java/com/example/filemanager/ListModel.kt

```diff
@@ -1,3 +1,7 @@
 package com.example.filemanager
 
-class ListModel(val fileName: String?) {}
+data class ListModel(
+    val icon: Int?,
+    val fileName: String?,
+    val additionalInfo: String?
```

Comment:
what is the `additionalInfo` used for?

Reply:
whould it possible to make all these fields non-nullable?

Reply:
I added the additionalInfo field for showing the details about the folder or file, maybe the size o the file or the nr of files in the folder, we need to discuss that, because it's a new idea that appeared during the process of analyzing other similar apps

Reply:
Yes, it is possible to make all fields non-nullable

Reply:
if `additionalInfo` is not used then I guess you can remove it and will add it back only when needed.